packages/alice_objectbox/lib/model/cached_alice_http_response.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:convert' show jsonDecode, jsonEncode;
+
+import 'package:alice/model/alice_http_response.dart';
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class CachedAliceHttpResponse implements AliceHttpResponse {
+  CachedAliceHttpResponse({
+    this.objectId = 0,
+    this.status = 0,
+    this.size = 0,
+    DateTime? time,
+    this.body,
+    this.headers,
+  }) : time = time ?? DateTime.now();
+
+  @Id()
+  int objectId;
+
+  @override
+  int? status;
+
+  @override
+  int size;
+
+  @override
+  @Property(type: PropertyType.dateNano)
+  DateTime time;
+
+  @override
+  @Transient()
+  dynamic body;
```

Comment:
Do we want to keep body which is large (i.e. few megabytes)?

Reply:
Hold on, I thought keeping the body is the point of this exercise. How else can you then check if the server returned garbage data?